oak_client/src/lib.rs

```diff
@@ -41,48 +39,36 @@ where
     T: micro_rpc::AsyncTransport<Error = anyhow::Error>,
 {
     transport: T,
-    crypto_provider: SenderCryptoProvider,
+    enclave_encryption_public_key: Vec<u8>,
```

Comment:
I think we should refer to this as server instead of enclave. The server may or may not be an enclave, and long term even the client may be an enclave actually. Also it pairs with this being an OakClient struct, I think.

Reply:
Done